### ISSUE TITLE
Pseudonymization of sensitive data [3+2]

### ISSUE THREAD

**Issue by MateStrysewskeSym** [hzi-braunschweig/SORMAS-Project#2130]
Prerequisite: #1158

NEW USER RIGHTS: SEE_SENSITIVE_DATA_IN_JURISDICTION & SEE_SENSITIVE_DATA_OUTSIDE_JURISDICTION
=> Prevent users WITHOUT this right from seeing:
  * Tasks assigned to users on district level or below (Community, Health facility, point of entry)
  * User names of personnel on district level or below (username, first & lastname)
  * Any location information below district level (e.g. the facility of a case)
  * Names of laboratories where samples have been sent to
  * All fields that allow users to type in free text
  * Detailed information about the employee status of cases or contacts

The default should probably be that every user has the SEE_SENSITIVE_DATA_IN_JURISDICTION user right and nobody has the SEE_SENSITIVE_DATA_OUTSIDE_JURISDICTION user right. German servers will need to be manually adjusted.

### Review
@MartinWahnschaffeSymeda

**Comment by MateStrysewskeSym** [comment]
New input from HZI:

- [ ] Pseudonymised should not just be emptied; instead, we should display a placeholder text that has a different style than normal text (gray-ish, maybe italic if that doesn't look too bad).
- [ ] For maps (dashboard & statistics), we can't remove the GPS coordinates completely because the cases still need to be displayed there. Instead, we need to keep them, but randomize the last parts so that the accuracy is reduced to an area of about 5 kilometers
- [ ] Don't pseudonymize the Postal Code completely; instead, reduce it to the first three digits

**Comment by MateStrysewskeSym** [comment]
Things that are probably out of scope for this issue but should be done asap in a follow-up issue:

* It's possible to search for confidential data; e.g. if I have access to a case but not to its personal details, I can still search for its name and get the case as a result
